pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -802,6 +826,17 @@ func (lrw *logicalReplicationWriterProcessor) flushBuffer(
 		stats.Add(lrw.bhStats[i])
 	}
 
+	if stats.debatchResolvedErrs > 0 {
+		// de-batching helped -- up the backoff.
```

Comment:
I'm not sure I understand this comment. That is, I see that we are increasing the backoff before we'll start batching again, but I'm unsure what we mean by `helped` here.